Cargo.toml

```diff
@@ -10,6 +10,7 @@ documentation = "https://docs.rs/colink"
 repository = "https://github.com/CoLearn-Dev/colink-sdk-rust-dev"
 
 [dependencies]
+async-recursion = "1.0.0"
```

Comment:
Don't forget to update the version number under `[package]` in this file (also in readme)